src/main/java/com/ays/admin_user/service/impl/AdminUserRegisterApplicationServiceImpl.java

```diff
@@ -16,4 +25,29 @@ public class AdminUserRegisterApplicationServiceImpl implements AdminUserRegiste
 
     private final AdminUserRegisterApplicationRepository adminUserRegisterApplicationRepository;
 
+
+    private final AdminUserRegisterApplicationEntityToAdminUserRegisterApplicationMapper adminUserRegisterApplicationEntityToAdminUserRegisterApplicationMapper = AdminUserRegisterApplicationEntityToAdminUserRegisterApplicationMapper.initialize();
+
+    /**
+     * Retrieves a paginated list of admin user registration applications based on the specified filters
+     *
+     * @param listRequest covering page, pageSize and filters
+     * @return a paginated list of admin user registration applications
+     */
+    @Override
+    public AysPage<AdminUserRegisterApplication> getRegistrationApplications(AdminUserRegisterApplicationListRequest listRequest) {
+        final Specification<AdminUserRegisterApplicationEntity> filters = listRequest
+                .toSpecification(AdminUserRegisterApplicationEntity.class);
+
+        final Page<AdminUserRegisterApplicationEntity> registerApplicationEntities = adminUserRegisterApplicationRepository
+                .findAll(filters, listRequest.toPageable());
+
+        final List<AdminUserRegisterApplication> registerApplications = adminUserRegisterApplicationEntityToAdminUserRegisterApplicationMapper.map(registerApplicationEntities.getContent());
+
+        return AysPage.of(
+                listRequest.getFilter(),
+                registerApplicationEntities,
+                registerApplications
+        );
+    }
```

Comment:
Sıralamayı nasıl yaptık burada?

Reply:
`toPagable` methodunda sort bulunuyor.

![image](https://github.com/afet-yonetim-sistemi/ays-be/assets/58088365/8421ff25-a83a-4504-a97f-68c03eec51ec)

Reply:
Alt yapıda her şey düşünülmüş 😄